src/ResourceManager/Resources/Commands.Resources/packages.config

```diff
@@ -1,6 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Azure.Management.Authorization" version="2.6.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ManagementPartner" version="1.0.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Authorization" version="2.6.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
Please remove the `Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure`, and `Newtonsoft.Json` files as these are always included in Commands.ResourceManager.Common project.